src/python_testing/TC_SEPRTestBase.py

```diff
@@ -0,0 +1,195 @@
+
+        logger.info(
+            f"  Component: price: {struct.price} source: {struct.source}, desc: {struct.description} tariffComponentID: {struct.tariffComponentID}")
+
+    async def send_get_detailed_price_request(self, endpoint: int = None, details: Clusters.CommodityPrice.Bitmaps = 0,
+                                              timedRequestTimeoutMs: int = 3000,
+                                              expected_status: Status = Status.Success):
+        try:
+            result = await self.send_single_cmd(cmd=Clusters.CommodityPrice.Commands.GetDetailedPriceRequest(
+                details=details),
+                endpoint=endpoint,
+                timedRequestTimeoutMs=timedRequestTimeoutMs)
+
+            return result
+
+        except InteractionModelError as e:
+            asserts.assert_equal(e.status, expected_status,
+                                 "Unexpected error returned")
+
+    async def send_get_detailed_forecast_request(self, endpoint: int = None, details: Clusters.CommodityPrice.Bitmaps = 0,
+                                                 timedRequestTimeoutMs: int = 3000,
+                                                 expected_status: Status = Status.Success):
+        try:
+            result = await self.send_single_cmd(cmd=Clusters.CommodityPrice.Commands.GetDetailedForecastRequest(
+                details=details),
+                endpoint=endpoint,
+                timedRequestTimeoutMs=timedRequestTimeoutMs)
+
+            return result
+
+        except InteractionModelError as e:
+            asserts.assert_equal(e.status, expected_status,
+                                 "Unexpected error returned")
+
+    async def send_test_event_trigger_price_update(self):
+        await self.send_test_event_triggers(eventTrigger=0x0095000000000000)
+
+    async def send_test_event_trigger_forecast_update(self):
+        await self.send_test_event_triggers(eventTrigger=0x0095000000000001)
+
+    def get_current_time_as_epoch_s(self):
+        """Returns current time in UTC in Matter Epoch_S"""
+        # Matter epoch is 0 hours, 0 minutes, 0 seconds on Jan 1, 2000 UTC
+        now_time = datetime.now()     # Get time in local time
+
+        # Shift to UTC so we can use timezone aware subtraction from Matter epoch in UTC
+        now_time = now_time.astimezone(timezone.utc)
+
+        matter_base_time = datetime(2000, 1, 1, 0, 0, 0, 0, tzinfo=timezone.utc)
+
+        now_time_delta = now_time - matter_base_time
+
+        now_time_epoch_s = int(now_time_delta.total_seconds())
+        return now_time_epoch_s
+
+    def convert_epoch_s_to_time(self, epoch_s, tz=timezone.utc):
+        if epoch_s is not NullValue:
+            delta_from_epoch = timedelta(seconds=epoch_s)
+            matter_epoch = datetime(2000, 1, 1, 0, 0, 0, 0, tz)
+
+            return matter_epoch + delta_from_epoch
+        else:
+            return "None"
```

Comment:
Return type is odd: string or datetime. Should this return `None` instead (not a string)?